release-notes.md

```diff
@@ -51,6 +51,14 @@ This document contains release notes per zib, indicating differences with their
 * The extension EpisodOfCare-Title has been replaced by ext-EpisodeOfCare.EpisodeOfCareName because this zib concept is functionally equivalent.
 * The extensions EpisodeOfCare-DateFirstEncounter and EpisodeOfCare-DateLastEncounter from the previous profile are not inherited because no functional backing exists.
 
+# zib-FLACCpainScale
+* The system and code value has changed to `http://snomed.info/sct` and `108291000146105` on `Observation.code`.
```

Comment:
Would be nice to know the original code. Moreover, the system nd code do not have to be in code syntax " ' ". The code value can be italic.

Reply:
Fixed